hypershift-operator/controllers/hostedcluster/internal/platform/aws/aws.go

```diff
@@ -31,6 +32,14 @@ const (
 	ImageStreamCAPA = "aws-cluster-api-controllers"
 )
 
+var (
+	SkippedCloudResourcesDeletionName = "hypershift_cluster_skipped_cloud_resources_deletion"
```

Comment:
please move this metric definition along with everything else in hostedcluster/metrics

Reply:
I will create 2 new packages, utils and metrics and move the things over to make this work in the same way as it is now. And I will need to duplicate a variable in order avoid to create another package (types) which will hold the types and constants of the hostedcluster package.

This will be better to do it in a follow up refactor PR (I've added a comment in the code for that)